torch_xla/csrc/ops/native_dropout.cpp

```diff
@@ -0,0 +1,39 @@
+#include "torch_xla/csrc/ops/native_dropout.h"
+
+#include "torch_xla/csrc/helpers.h"
+#include "torch_xla/csrc/lowering_context.h"
+#include "torch_xla/csrc/xla_lower_util.h"
+
+namespace torch_xla {
+namespace {
+
+xla::Shape NodeOutputShape(const torch::lazy::Value& input) {
+  xla::Shape input_shape = GetXlaShape(input);
```

Comment:
I cannot repro the CI failure on native_dropout_backward. But it seems type of mask output mismatches with the type specified here.

Shall we just do the following here.
```
return xla::ShapeUtil::MakeTupleShape({input_shape, input_shape});
```
I had a try locally, the torch tensor would still have boolean type, seems the dtype would be casted when the tensor is built from lazy ir.

Reply:
Yes, the conversion here doesn't affect type.